shared/src/commonMain/kotlin/org/hyperskill/app/progresses/domain/repository/ProgressesRepository.kt

```diff
@@ -17,10 +17,12 @@ interface ProgressesRepository {
             getTopicsProgresses(listOf(topicId)).getOrThrow().first()
         }
 
-    suspend fun getProjectsProgresses(projectsIds: List<Long>): Result<List<ProjectProgress>>
+    suspend fun getProjectsProgresses(projectsIds: List<Long>, forceLoadFromRemote: Boolean): Result<List<ProjectProgress>>
 
-    suspend fun getProjectProgress(projectId: Long): Result<ProjectProgress> =
+    suspend fun getProjectProgress(projectId: Long, forceLoadFromRemote: Boolean): Result<ProjectProgress> =
         kotlin.runCatching {
-            getProjectsProgresses(listOf(projectId)).getOrThrow().first()
+            getProjectsProgresses(listOf(projectId), forceLoadFromRemote).getOrThrow().first()
         }
+
+    fun clearCache()
```

Comment:
This function isn't used, we're clearing cache after deauthorization in `SingletonRepositoriesComponent`